drive_node:
  ros__parameters:
    wheels:
      separation: 0.244
      radius: 0.0405

    serial:
      port: "/dev/ACM0"
      baudrate: 115200

    joint_states_frame: "base_footprint"
    odom_frame: "odom"
    base_frame: "base_footprint"
